Packs/ExportIndicators/Integrations/ExportIndicators/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+  - Added the *offset* parameter to the ***eis-update*** command.
+  - Added support for the following inline URL parameters.
+    - n - The number of indicators to fetch.
+    - s - The first index from which to fetch indicators.
+    - v - The output format for indicators.
+    - q - The query that defines which indicators to fetch.
 
```

Comment:
This is an extensive addition, this should be documented in the readme. Please add it there as well as to the detailed description, as this is not trivial for a user with no past experience with these arguments.

Reply:
done